src/component/board/board.component.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { StyledBoard } from './board.style'
+import Slot from '../slot'
+
+const Board = () => {
+  return (
+    <StyledBoard side={350}>
```

Comment:
Is it "side" or "size"?

Reply:
It is "side" :)

Reply:
@marianzburlea we probably did a mistake naming this property "side" :-?